packages/contracts/deploy-config/devnetL1.json

```
@@ -24,16 +25,16 @@
   "finalSystemOwner": "0xBcd4042DE499D14e55001CcbB24a551F3b954096",
   "portalGuardian": "0xBcd4042DE499D14e55001CcbB24a551F3b954096",
   "controller": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
-  "finalizationPeriodSeconds": 2,
+  "finalizationPeriodSeconds": 600,
   "deploymentWaitConfirmations": 1,
   "fundDevAccounts": true,
   "l2GenesisBlockBaseFeePerGas": "0x3B9ACA00",
   "gasPriceOracleOverhead": 2100,
-  "gasPriceOracleScalar": 1000000,
+  "gasPriceOracleScalar": 1500000,
   "eip1559Denominator": 8,
   "eip1559Elasticity": 2,
   "l1GenesisBlockTimestamp": "0x638a4554",
-  "l1StartingBlockTag": "earliest",
+  "colosseumCreationPeriod": 500,
```

It would be nice to have a unified way of attaching units to variable names.
Like `finalizationPeriodSeconds` above.
```suggestion
  "colosseumCreationPeriodSeconds": 500,
```